bin/php-validate

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+phpcs --standard="$PWD/vendor/pixelunion/php-validate/config/laravel_phpcs_ruleset.xml" $1
```

Comment:
This way of getting to the config file feels weird – I think this is probably "the way" to do this with a shell script, but it seems _funny_.

Can this be done with a post install hook in the composer package that does something with this?
https://github.com/squizlabs/PHP_CodeSniffer/wiki/Configuration-Options#setting-the-default-coding-standard

Reply:
going to look into this as a future PR 👍